sdk/storage/azqueue/queue/client.go

```diff
@@ -0,0 +1,148 @@
+package queue
```

Comment:
These files would go directly in the `azqueue` package, no need for a `queue` package.

Also, this file, and `models.go`, are missing the build constraint and copyright header comments.